wallet-new/src/Cardano/Wallet/Kernel/DB/Resolved.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE DeriveGeneric, DeriveDataTypeable             #-}
```

Comment:
Why do we need this one? This PR doesn't add any extra generic derivation, I'm confused 😀

Reply:
that was shrapnel - removed